pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +27,19 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+     
+     JSONArray ja = new JSONArray();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Comment prueba = new Comment("Title", "Msg");
+     JSONObject jo = new JSONObject();
```

Comment:
You can achieve this same thing with GSON#toJson like this example https://github.com/googleinterns/step/blob/master/walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java#L71-L75

Reply:
+1. Sorry for the advice during our 1:1 Pamela, I didn't know that Gson was already imported for you. It seems like this is what they want you to do. Also, good job figuring out how to import this in pom.xml!

Reply:
Okay no worries, I think it's done and thank you!